src/lib/ecore_wl2/ecore_wl2_input.c

```diff
@@ -43,6 +43,10 @@
 #include <sys/mman.h>
 #include "ecore_wl2_private.h"
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)